### PR TITLE
Minor tweaks for grammar

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

File: vipps-qr-api.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ Body:
 }
 ```
 
-The response will be similar to this, where the URL in the responseBody will be a link to the image as defined in the accept header:
+The response will be similar to this, where the URL in the `responseBody` will be a link to the image as defined in the accept header:
```
Comment (okpedersen) [inline vipps-qr-api.md:174]: maybe "response body" or "body of the response"?

File: vipps-qr-api.md [hunk 2]
```diff
@@ -227,7 +227,7 @@ QR code has been scanned). Once this field is set, you can safely show a
 
 Merchant redirect QR codes allows you to make printable QR
 codes that redirect the user to your webpage. This can be used for one-offs,
-such as tv-commercials; as well as for permanent use cases, such as stickers,
+such as tv commercials; as well as for permanent use cases, such as stickers,
```
Comment (okpedersen) [inline vipps-qr-api.md:230]: tv => TV?

Comment (rebekaburnett) [inline vipps-qr-api.md:230]:
```suggestion
such as TV commercials; as well as for permanent use cases, such as stickers,
```

**Review by okpedersen — APPROVED** [review]

File: vipps-qr-api.md (hunk 1 above)
Comment (rebekaburnett) [reply vipps-qr-api.md]: Changed to   "body of the response"

**Pull request merged**